src/VersionsCheck.php

```diff
@@ -101,7 +101,13 @@ public function versionCompare($version1, $operator, $version2)
 
     private function createNotUpToDateOutput(array &$output)
     {
-        $output[] = '<warning>Some packages are not up to date:</warning>';
+        $totalOutdatedPackages = count($this->outdatedPackages);
```

Comment:
* `$outdatedPackagesCount`
* I think the newline can be removed, it's for the same logic. :+1: